paper-radio-button.html

```diff
@@ -74,6 +74,7 @@
       #radioContainer {
         @apply(--layout-inline);
         @apply(--layout-center-center);
+        flex-shrink: 0;
```

Comment:
By default, `#radioContainer`'s parent (`:host`) is not a flex container, so unless you're making it one (e.g. `paper-radio-button { display: flex }`), `#radioContainer` should not shrink.